src/main/java/com/disi/TravelPoints/controller/EmailController.java

```diff
@@ -0,0 +1,50 @@
+package com.disi.TravelPoints.controller;
+
+
+
+import com.disi.TravelPoints.Email.Email;
+import com.disi.TravelPoints.dto.LandmarkDetails;
+import com.disi.TravelPoints.dto.AddLandmarkRequest;
+import com.disi.TravelPoints.exception.CustomException;
+import com.disi.TravelPoints.service.LandmarkService;
+import lombok.*;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import java.util.Properties;
+import javax.mail.*;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+@RestController
+@RequestMapping("api/v1/email")
+@RequiredArgsConstructor
+public class EmailController {
+
+    @PostMapping
```

Comment:
I think there shouldn't be so much code in a controller. Please move the logic to a service.

Reply:
Yes, I think it would be better. Please create one.